recipe/meta.yaml

```
@@ -10,7 +10,7 @@ source:
   sha256: cd1f3ad3af9dbff33aef9e7406d2130d624a0bd1d5aa5970fc713a6421165a4e
 
 build:
-  number: 0
+  number: 1
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

This will help to bring some of these build issues to light sooner.

I tried to add this and the suggested fix myself, but I realize that this is one of the repos not under @conda-forge/scikit-hep maintenance yet (so I created Issue #6 and PR #7)